src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmClrRuntimeInstance.cs

```diff
@@ -47,10 +47,7 @@ internal DkmClrRuntimeInstance(Assembly assembly) : this(new[] { assembly })
             bool enableNativeDebugging = false)
             : base(enableNativeDebugging)
         {
-            if (getModule == null)
-            {
-                getModule = (r, a) => new DkmClrModuleInstance(r, a, (a != null) ? new DkmModule(a.GetName().Name + ".dll") : null);
-            }
+            getModule ??= (r, a) => new DkmClrModuleInstance(r, a, (a != null) ? new DkmModule(a.GetName().Name + ".dll") : null);
```

Comment:
ahh the old null coalescing lambda with a ternary.. if i had a dollar for every time i've seen that!

Reply:
tis indeed glorious.  bask in it and feel the warmth shine on your face.